test/rest.test.js

```diff
@@ -319,89 +319,34 @@ describe('strong-remoting-rest', function() {
       );
     });
 
-    it('should support cors', function(done) {
+    it('should reject cross-origin requests', function(done) {
       request(app).post(method.url)
         .set('Accept', 'application/json')
         .set('Content-Type', 'application/json')
         .set('Origin', 'http://localhost:3001')
         .send({ person: 'ABC' })
-        .expect('Access-Control-Allow-Origin', 'http://localhost:3001')
-        .expect('Access-Control-Allow-Credentials', 'true')
-        .expect(200, done);
-    });
-
-    it('should skip cors if origin is the same as the request url', function(done) {
-      var server = request(app).post(method.url);
-      var url = server.url.replace('/testClass/testMethod', '');
-      server
-        .set('Accept', 'application/json')
-        .set('Content-Type', 'application/json')
-        .set('Origin', url)
-        .send({ person: 'ABC' })
-        .end(function(err, res) {
-          assert(res.get('Access-Control-Allow-Origin') === undefined);
-          assert(res.get('Access-Control-Allow-Credentials') === undefined);
+        .expect(200, function(err, res) {
```

Comment:
In my understanding of CORS, it's the responsibility of the browser (the client) to enforce cross-origin policies. The server is required to describe which origins are allowed to access it from the browser.

See https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS (emphasis is mine)
> The Cross-Origin Resource Sharing standard works by adding new HTTP headers that **allow servers to describe the set of origins that are permitted to read that information using a web browser**.  Additionally, for HTTP request methods that can cause side-effects on user data (in particular, for HTTP methods other than GET, or for POST usage with certain MIME types), the specification mandates that browsers "preflight" the request, soliciting supported methods from the server with an HTTP OPTIONS request method, and then, upon "approval" from the server, sending the actual request with the actual HTTP request method.  Servers can also notify clients whether "credentials" (including Cookies and HTTP Authentication data) should be sent with requests.

In our tests here, we are not acting like a browser (the `supertest` library is an HTTP agent only, not a browser-like client) and therefore the status code `200` is expected.